src/styles/index.scss

```diff
@@ -20,6 +20,53 @@ body {
   pointer-events: none;
 }
 
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
+  max-height: 58px;
+  transition: max-height 0.3s ease-in-out;
+}
```

Comment:
nesting?

Reply:
These styles were given in the task for the TransitionGroup